routes/movies.routes.js

```diff
@@ -0,0 +1,115 @@
+const { populate } = require("../models/Celebrity.model");
+const Celebrity = require("../models/Celebrity.model");
+const Movie = require("../models/Movie.model");
+
+const router = require("express").Router();
+
+router.get("/movies/create", (req, res) => {
+
+
+    Celebrity.find()
+        .then(celebritiesArr => {
+            res.render("movies/new-movie", { celebritiesArr });
+        })
+        .catch((error) => {
+            console.log("Error getting celebrities from DB", error);
+            next(error);
+        })
+})
+
+router.post("/movies/create", (req, res) => {
+    const movieDetails = {
+        title: req.body.title,
+        genre: req.body.genre,
+        plot: req.body.plot,
+        cast: req.body.cast,
+    };
+
+    Movie.create(movieDetails)
+        .then(() => {
+            res.redirect("/movies");
+        })
+        .catch((error) => {
+            console.log("Error creating book in the DB", error);
+            next(error);
+        })
+})
+
+router.get("/movies", (req, res, next) => {
+
+    Movie.find()
+        .populate('cast')
+        .then((moviesFromDB) => {
+            const data = {
+                moviesArr: moviesFromDB
+            };
+
+
+            res.render("movies/movies", data);
+        })
+
+        .catch((error) => {
+            console.log("Error getting data from DB", error)
+            next(error);
+        })
+});
+
+router.get("/movies/:id", (req, res) => {
+    const id = req.params.id;
```

Comment:
This works, but you could also destructure the variable, meaning that it could also look like this:
```
const {id} = req.params;
```